src/tests/api.test.js

```diff
@@ -150,9 +149,10 @@ describe('API test', () => {
     init(configuration, onMessage).then(({ botName, history }) => {
       expect(botName).toEqual('Templates');
       expect(history).toEqual([
```

Comment:
use mock local storage here for history?

Reply:
Can you clarify on this? This history is just local array returning from init call. Internally, mock-local-storage is used for storing chatHistory.